footprints/main/views.py

```diff
@@ -534,3 +543,44 @@ def form_valid(self, form):
         send_mail(subject, tmpl.render(Context(form_data)), sender, recipients)
 
         return super(ContactUsView, self).form_valid(form)
+
+
+class SignS3View(LoggedInMixin, View):
```

Comment:
Nice. I was going to suggest packaging it up. Noting a change in Footprints: The view is using the STORAGES bucket/uploads (a la a pagetree app) rather than a separate bucket. A dev bucket is configured by default in settings_shared.